app/src/main/java/org/eyeseetea/malariacare/data/database/datasources/MediaDataSource.java

```diff
@@ -0,0 +1,28 @@
+package org.eyeseetea.malariacare.data.database.datasources;
+
+import org.eyeseetea.malariacare.data.database.model.MediaDB;
+import org.eyeseetea.malariacare.domain.boundary.repositories.IMediaRepository;
+import org.eyeseetea.malariacare.domain.entity.Media;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MediaDataSource implements IMediaRepository {
```

Comment:
@manuelplazaspalacio we had talked on some pair programming with @idelcano avoid create repository interface and the implementation as a data source. We had talked from now create always repository interface and the implementation as a repository and create LocalDataSource , can you refactor it?